.github/actions/setup-jdks/action.yml

```diff
@@ -20,7 +20,8 @@ runs:
     - name: 'Set up JDK 8'
       uses: actions/setup-java@v4
       with:
-        distribution: 'temurin'
+        # Temurin JDK 8 for macos on ARM is not available: https://github.com/adoptium/adoptium/issues/96
+        distribution: 'zulu'
```

Comment:
CoPilot suggests this as a solution for @Vampire's comment. Of course, it can only be tested by applying the suggestion.

```suggestion
        distribution: ${{ runner.os == 'macOS' && 'zulu' || 'temurin' }}
```